### ISSUE TITLE
sync block error with height 9073579

### ISSUE THREAD

**Issue by liuxh-go** [BuildOnViction/victionchain#317]
* code version:v2.2.4

* error log
```
ERROR[10-16|09:32:37]
########## BAD BLOCK #########
Chain config: {ChainID: 88 Homestead: 1 DAO: <nil> DAOSupport: false EIP150: 2 EIP155: 3 EIP158: 3 Byzantium: 4 Constantinople: <nil> Engine: posv}

Number: 9073579
Hash: 0xe5d21d25a036d0f78263b4a762a4c2902960f87d06603174a075d4a9840b512d


Error: insufficient balance for transfer
##############################

WARN [10-16|09:32:37] Synchronisation failed, dropping peer    peer=5b65023779f99a6e err="retrieved hash chain is invalid"
```

**Comment by liuxh-go** [comment]
Can someone help me?

**Comment by thanhson1085** [comment]
You should download chaindata (tomo, tomox) snapshot from https://tomotools.com/ to fix the issue

**Comment by thanhson1085** [comment]
Fixed in v2.3.0
